packages/block-editor/src/components/off-canvas-editor/index.js

```diff
@@ -104,9 +107,9 @@ function __ExperimentalOffCanvasEditor(
 		setExpandedState,
 	} );
 	const selectEditorBlock = useCallback(
-		( event, clientId ) => {
-			updateBlockSelection( event, clientId );
-			setSelectedTreeId( clientId );
+		( event, _clientId ) => {
```

Comment:
Can we not go for the underscore? Instead name the clientId more specifically maybe? Like selectedBlockClientId, and same for the rest of `_clientId`.

Reply:
✅